fsspec/implementations/smb.py

```diff
@@ -83,13 +84,20 @@ def __init__(
             Port to connect with. Usually 445, sometimes 139.
         username: str or None
             Username to connect with. Required if Kerberos auth is not being used.
-        password: str of None
+        password: str or None
             User's password on the server, if using username
         timeout: int
             Connection timeout in seconds
         encrypt: bool
             Whether to force encryption or not, once this has been set to True
             the session cannot be changed back to False.
+        share_access: str or None
+            specifies the type of access that is allowed when a handle to this 
+            file is opened by another process.
```

Comment:
The wording here is a little unclear - we are defining what the default share access will be for any open-file operations, unless otherwise specified.

Reply:
Yes you're right. I've reworded it.